src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryService.scala

```diff
@@ -23,25 +24,27 @@ import scala.concurrent.{ExecutionContext, Future}
 object LibraryService {
   sealed trait LibraryServiceMessage
   case class UpdateAttributes(ns: String, name: String, attrs: JsValue) extends LibraryServiceMessage
+  case object IndexAll extends LibraryServiceMessage
 
   def props(libraryServiceConstructor: UserInfo => LibraryService, userInfo: UserInfo): Props = {
     Props(libraryServiceConstructor(userInfo))
   }
 
   def constructor(app: Application)(userInfo: UserInfo)(implicit executionContext: ExecutionContext) =
-    new LibraryService(userInfo, app.rawlsDAO)
+    new LibraryService(userInfo, app.rawlsDAO, app.searchDAO)
 }
 
 
-class LibraryService (protected val argUserInfo: UserInfo, val rawlsDAO: RawlsDAO)(implicit protected val executionContext: ExecutionContext) extends Actor
-  with LibraryServiceSupport with RoleSupport with SprayJsonSupport {
+class LibraryService (protected val argUserInfo: UserInfo, val rawlsDAO: RawlsDAO, val searchDAO: SearchDAO)(implicit protected val executionContext: ExecutionContext) extends Actor
+  with LibraryServiceSupport with RoleSupport with SprayJsonSupport with LazyLogging {
 
   lazy val log = LoggerFactory.getLogger(getClass)
 
   implicit val userInfo = argUserInfo
 
   override def receive = {
-    case UpdateAttributes(ns: String, name: String, attrs: JsValue) => asCurator {updateAttributes(ns, name, attrs)} pipeTo sender
+    case UpdateAttributes(ns: String, name: String, attrs: JsValue) => asCurator {updateAttributes(ns, name, attrs)} pipeTo context.parent
+    case IndexAll => asAdmin {indexAll pipeTo context.parent}
```

Comment:
I'm not certain why `sender` works in rawls, but orch requires `context.parent`. But orch *definitely* needs `context.parent`; messages were getting dropped.